### ISSUE TITLE
Can't bencode torrent file

### ISSUE THREAD

**Issue by vinz243** [whtsky/bencoder.pyx#1]
Hi,
I have a torrent file decoded, and now I need to bencode it again after having modified a few things. However, I get an error:

```
Traceback (most recent call last):
  File "test.py", line 61, in <module>
    print (bencode(data['info']))
  File "bencoder.pyx", line 159, in bencoder.bencode (bencoder.c:4217)
  File "bencoder.pyx", line 100, in bencoder.encode (bencoder.c:2787)
  File "bencoder.pyx", line 141, in bencoder.encode_dict (bencoder.c:4084)
  File "bencoder.pyx", line 100, in bencoder.encode (bencoder.c:2787)
  File "bencoder.pyx", line 109, in bencoder.encode_int (bencoder.c:3015)
OverflowError: value too large to convert to int
```

Here is a pprint of the data I'm trying to encode:
https://gist.github.com/vinz243/9071201b4057f6f838e97936595572fd

Thanks in advance

**Comment by whtsky** [comment]
Fixed in v1.1.2

**Comment by vinz243** [comment]
Thanks a lot :)
